### DISCUSSION TITLE
`Text` type on Substrate

### DISCUSSION THREAD

**Discussion by VictorVicente** [polkascan/py-scale-codec#98] [Q&A]
Hey, I found this great library and I'm trying to adapt it with to our substrate chain Polymesh.
I'm facing some troubles, maybe you could help me:
1. `Text` type from `schema.json` in Substrate: How should I represent it? Should it be any kind of `Vec<>`?
2. It doesn't recognize `[u8; 32]` on my json. Even when it is present on the `default.json` file and it's working fine there. Is there anything else I should add to make it work? 

Thanks in advance!

**Comment by arjanz** [comment]
1. You can use `Bytes` for a `Text` type, our codec doesn't make that distinction (yet).

2. That is odd, `[u8; 32]` should match the `VecU8Length32` decoding class, did you check for spaces etc?
